src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
concerning my first question: we have initial code like this:
```
col_missions["Assign Scavenging Patrol"] = "line1 \n"
"line2 \n"
"lineN";
```
is it okay to apply only one set of parentheses like it done here:
```
col_missions["Assign Scavenging Patrol"] = _( "line1 \n"
"line2 \n"
"lineN" );
```
or we need to put pair of parentheses to each line?